### PR TITLE
added 2 more sdk update dependent repositories.

### PR DESCRIPTION
**Pull request by shahnawaz-creator**
### Changes
-

### Fixes
-

### Tests
Tasks to complete before merging PR:
- [ ]  Ensure system tests are passing. If not [Run them manually](https://github.com/0chain/gosdk/actions/workflows/system_tests.yml) to check for any regressions :clipboard:
- [ ]  Do any new system tests need added to test this change? do any existing system tests need updated? If so create a PR at [0chain/system_test](https://github.com/0chain/system_test)
- [ ]  Merge your system tests PR to master AFTER merging this PR

### Associated PRs (Link as appropriate):
- blobber:
- 0chain:
- system_test:
- zboxcli:
- zwalletcli:
- Other: ...

### CODE REVIEW

**Review by dabasov — DISMISSED** [review]
lgtm

File: .github/workflows/sdk-release.yml [hunk 1]
```diff
@@ -93,7 +93,7 @@ jobs:
       fail-fast: false
       max-parallel: 1
       matrix:
-        repo: [block-explorer, 0nft, 0box, 0dns, 0proxy, zwalletcli, zboxcli, zboxmobile, blobber ]
+        repo: [block-explorer, 0nft, 0box, 0dns, 0proxy, zwalletcli, zboxcli, zboxmobile, blobber, s3server, s3-migration ]
```
Comment (cnlangzi) [inline .github/workflows/sdk-release.yml]: block-explorer, 0proxy and zboxmobile can be removed now.

Comment (cnlangzi) [reply .github/workflows/sdk-release.yml]: s3server should be `zs3server`. and please add https://github.com/0chain/zboxcli/blob/staging/.github/workflows/gosdk.yml on zs3server and s3-migration repos.

**Review by cnlangzi — CHANGES_REQUESTED** [review]
@shahnawaz-creator  please fix them first. thanks.

File: .github/workflows/sdk-release.yml (hunk 1 above)
Comment (shahnawaz-creator) [reply .github/workflows/sdk-release.yml]: fixed

**Review by cnlangzi — APPROVED** [review]
@shahnawaz-creator  please don't forget to add [create-pr](https://github.com/0chain/zboxcli/blob/staging/.github/workflows/gosdk.yml) workflow on zs3server and s3migration.

File: .github/workflows/sdk-release.yml [hunk 2]
```diff
@@ -93,7 +93,7 @@ jobs:
       fail-fast: false
       max-parallel: 1
       matrix:
-        repo: [block-explorer, 0nft, 0box, 0dns, 0proxy, zwalletcli, zboxcli, zboxmobile, blobber ]
+        repo: [0nft, 0box, 0dns, zwalletcli, zboxcli, blobber, zs3server, s3-migration ]
```
Comment (boddumanohar) [inline .github/workflows/sdk-release.yml]: gosdk is also used in [authorizers](https://github.com/0chain/token_bridge_authserver/blob/staging/code/go/0chain.net/authorizer/go.mod#L6)

**Review by boddumanohar — APPROVED** [review]
also used in authorizers

File: .github/workflows/sdk-release.yml (hunk 2 above)
Comment (cnlangzi) [reply .github/workflows/sdk-release.yml]: web-apps should be upgraded. @ranajahanzaib  could you copy it from block-explorer?

Comment (shahnawaz-creator) [reply .github/workflows/sdk-release.yml]: added `token_bridge_authserver` repo to update gosdk

**Pull request merged**